package.json

```diff
@@ -15,26 +15,29 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/autotelic/fastify-plugin-template.git"
+    "url": "git+https://github.com/autotelic/fastify-stream-to-csv.git"
```

Comment:
Can we remove the postinstall script? It causes issues with installation of the packages. It should also be removed from the template repo.

https://github.com/autotelic/fastify-stream-to-csv/pull/2/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R14